File: src/Clients/accountant2/src/lib/models/entities/debt.ts
import type { Syncable } from '$lib/models/syncable';

export class DebtModel implements Syncable {
	synced = false;

	constructor(
		public id: number,
		public person: string,
		public amount: number,
		public currency: string,
		public description: string | null,
		public userIsDebtor: boolean,
		public createdDate: Date | null,
		public modifiedDate: Date | null
	) {}
}

File: src/Application/Domain/Accountant/Debt.cs
namespace Application.Domain.Accountant;

public class Debt : Entity
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public string Person { get; set; }
    public decimal Amount { get; set; }
    public string Currency { get; set; }
    public bool UserIsDebtor { get; set; }
    public string Description { get; set; }
}
